tests/system/config/mysqlbeat.yml.j2

```diff
@@ -14,12 +14,12 @@ output:
     enabled: true
 
     # Path to the directory where to save the generated files. The option is mandatory.
-    path: {{ output_file_path|default(beat.working_dir + "/output") }}
+    path: {{ '{{' }} output_file_path|default(beat.working_dir + "/output") {{ '}}' }}
```

Comment:
Why are we escaping `{{` here? Isn't the purpose to actually evaluate and insert the value of `output_file_path|default(beat.working_dir + "/output")`? (comment applies to line 22 as well)

Reply:
@mdelaney it's autogenerated by libbeat 
link to similar at burrowbeat https://github.com/Tubular/burrowbeat/blob/develop/tests/system/config/burrowbeat.yml.j2#L30